service/lint.ml

```diff
@@ -1,18 +1,28 @@
 open Current.Syntax
 module Docker = Conf.Builder_amd1
 
-let format_dockerfile ~base ~ocamlformat_version =
-  let open Dockerfile in
-  from (Docker.Image.hash base)
-  @@ run "opam depext ocamlformat=%s" ocamlformat_version
-  @@ run "opam install ocamlformat=%s" ocamlformat_version
-  @@ copy ~chown:"opam" ~src:["./"] ~dst:"./src" ()
-  @@ workdir "src"
+type ocamlformat_version = [
+  | `Vendored
+  | `Version of string
+]
```

Comment:
Now that Analyse is a library, we could use it here (or just `Analyse_ocamlformat`).